### ISSUE TITLE
CCEffect: "Environment" connection can't be made when dragging to a node in the timeline

### ISSUE THREAD

**Issue by LearnCocos2D** [spritebuilder/SpriteBuilder#868]
The "Environment" property of the Refraction/Reflection/Glass (env) effect has a "drag a line to target node" circle like joints do.

However I can't drop that line onto a node in the timeline, it will highlight the node as valid drop target but it won't add the node as Environment target when letting go of the mouse button. It works when dropping the line onto the same node in the stage, the node is then correctly added as Environment reference.

Basically the problem is that the line-dragging will not accept the nodes in the timeline hierarchy as valid drop targets. This behavior is consistent for all four environment properties of the refraction, reflection and glass effect (two environments).

**Comment by vlidholt** [comment]
Duplicate of #844
